# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/moz.build
# Commit: 9b4ea6ecadd1
# Full Hash: 9b4ea6ecadd1e56b9a279feab220a8bfab36809c
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 16:14:38
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Directly use tx instead of the older avfft interface. r=karlt,media-playback-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D202156
# ==============================================================================

diff -r 7c71c3d1a738 -r 9b4ea6ecadd1 dom/media/platforms/moz.build
--- a/dom/media/platforms/moz.build	Wed Feb 28 12:50:27 2024 +0000
+++ b/dom/media/platforms/moz.build	Wed Feb 28 12:50:27 2024 +0000
@@ -48,10 +48,6 @@
     DIRS += ["wmf"]
 
 if CONFIG["MOZ_FFVPX"] or CONFIG["MOZ_FFMPEG"]:
-    # common code to either FFmpeg or FFVPX
-    EXPORTS += [
-        "ffmpeg/FFmpegRDFTTypes.h",
-    ]
     UNIFIED_SOURCES += [
         "ffmpeg/FFmpegLibWrapper.cpp",
     ]